dashboard/test/ui/features/feedback_tab.feature

```diff
@@ -56,19 +50,29 @@ Scenario: With dev flag, as teacher, tab is invisible when not reviewing student
   Then I click selector ".show-handle .fa-chevron-left"
   Then I click selector ".section-student .name a"
   And I press the first "#ui-test-feedback-input" element
+  And element "#ui-test-submit-feedback" contains text "Save and share"
+  And element "#ui-test-feedback-time" does not exist
   And I press keys "Nice!" for element "#ui-test-feedback-input"
   And I press "#ui-test-submit-feedback" using jQuery
   And I wait until ".editor-column" contains text "Nice!"
+  And element "#ui-test-feedback-time" contains text "Last updated"
+  And element "#ui-test-submit-feedback" contains text "Update"
 
   #As teacher, refresh page and latest feedback is visible
   And I reload the page
   And I wait for the page to fully load
   And I wait until ".editor-column" contains text "Nice!"
+  And element "#ui-test-feedback-time" contains text "Last updated"
+  And element "#ui-test-feedback-time" contains text "ago"
+  And element "#ui-test-submit-feedback" contains text "Update"
 
   #As student, latest feedback from teacher is displayed
   Then I sign out
   And I sign in as "Lillian"
   And I am on "http://studio.code.org/s/allthethings/stage/18/puzzle/7?enableExperiments=devCommentBoxTab"
   And I press the first ".uitest-feedback" element
   And I wait until ".editor-column" contains text "Nice!"
-  And I am on "http://studio.code.org/s/allthethings/stage/18/puzzle/7?disableExperiments=devCommentBoxTab"
+  And element ".editor-column" contains text "Feedback from Teacher_Lillian"
+  And element ".editor-column" contains text "(From"
+  And element ".editor-column" contains text "ago):"
```

Comment:
It'd be neat to write a step
```gherkin
And element ".editor-column" contains text matching /Feedback from Teacher_Lillian \(From .* ago\):/
```
Thoughts?

Reply:
Yeah! I think that'd be super useful. I'm going to take a stab at it.